.github/workflows/build.yml

```diff
@@ -33,6 +33,12 @@ jobs:
       - name: Build with cx_Freeze
         run: python torrent-client/setup.py build
 
+      - name: Install bsdtar
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libarchive-tools
```

Comment:
Electron builder documentation says to install `bdstar` package. Any reason why you need the package you added?

https://www.electron.build/multi-platform-build.html#linux

PS: I'm not a regular Linux user and I have zero experience with Arch

Reply:
Because bsdtar is a part of libarchive-tools. [There is no separate package for it on ubuntu](https://packages.ubuntu.com/search?keywords=bsdtar&searchon=names&suite=noble&section=all).

Maybe docs is a bit outdated

Reply:
> Because bsdtar is a part of libarchive-tools. [There is no separate package for it on ubuntu](https://packages.ubuntu.com/search?keywords=bsdtar&searchon=names&suite=noble&section=all).
> 
> Maybe docs is a bit outdated

This link is not working anymore. But I believe it is not possible to use the **apt** within arch, because arch does not use apt but pacman as default.

Reply:
![изображение](https://github.com/user-attachments/assets/06b97346-5678-4f36-baea-86363101b327)
It is still working for me. But anyway, there is no separate package for Ubuntu.

On Arch, it seems like it is preinstalled in some package, but it doesn't matter because all Linux packages are build on Ubuntu-latest